KubeArmor/core/containerdHandler.go

```diff
@@ -110,7 +114,27 @@ func NewContainerdHandler() *ContainerdHandler {
 	ch.docker = namespaces.WithNamespace(context.Background(), "moby")
 
 	// containerd namespace
-	ch.containerd = namespaces.WithNamespace(context.Background(), "k8s.io")
+	client, err := containerd.New(socketPath)
```

Comment:
Can we use the namespaces API client over the existing gRPC connection instead of creating a full-fledged client? We do the same for Containers and Tasks.
Ref - https://pkg.go.dev/github.com/containerd/containerd/api@v1.7.19/services/namespaces/v1

Reply:
okay sure will do that

Reply:
Have made the changes @DelusionalOptimist